roles/jd-server/jds-config.toml

```diff
@@ -4,16 +4,23 @@ authority_secret_key = "2Z1FZug7mZNyM63ggkm37r4oKQ29khLjAvEx43rGkFN47RcJ2t"
 cert_validity_sec = 3600
 test_only_listen_adress_plain =  "0.0.0.0:34250"
 listen_address = "0.0.0.0:34254"
-# list of compressed or uncompressed pubkeys for coinbase payout (only supports 1 item in the array at this point)
+
+# list of coinbase outputs used to build the coinbase tx
+# ! right now only one output is supported, so comment all the ones you don't need !
 coinbase_outputs = [
```

Comment:
change the invalid values in `output_script_value` and with `...`

Reply:
do you mean all the ones which are commented or only the P2SH and P2WSH which would require a script in the `output_script_value`?

Reply:
What I mean is that all the pair output_script_type,  output_script_value should be valid. If a script value is not valid just replace it with ...